src/Test/L0/Worker/TestResults/JunitResultReaderTests.cs

```diff
@@ -0,0 +1,326 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Xml;
+using Microsoft.VisualStudio.Services.Agent.Worker.TestResults;
+using Microsoft.VisualStudio.Services.Agent.Worker;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.VisualStudio.Services.Agent.Tests.Worker.TestResults
+{
+    [TestClass]
```

Comment:
We have been using Xunit for unit tests. Please take a look at the current tests.